app/views/kaui/refunds/_form.html.erb

```diff
@@ -158,8 +179,32 @@
         }
         recomputeRefundAmountAndValidateAmount();
         validateInvoiceItemAmount(id);
+        checkSelectAllCheckboxStatus();
     };
 
+    /*
+    * When clicking select all checkbox - select each items, disable amount and recompute total refund amount
+    */
+    var onClickInvoiceItemsSelectAll = function(event){
+        var isChecked = false;
+        if ($(this).is(':checked')) {
+            isChecked = true;
+        }
+        $('input').filter(function() { return this.id.match(/tf_adj_/) }).each(function() {
+            var textFieldId = this.id;
+            var checkboxId = textToCheckboxId(this.id);
+            if(checkboxId && textFieldId){
+                $("#" + checkboxId).prop('checked', isChecked);
+                $("#" + textFieldId).prop('readonly', isChecked);
```

Comment:
I guess to not to create new jQuery object repeatedly we can catch the object like
`var textField = $(this);` in the beginning of the function as use it like
```
textField.prop('readonly', isChecked);
        if (!isChecked) {
            textField.attr('value', textField.attr('originalValue'));
        }
```

Reply:
done

Reply:
Have you pushed it? Because I still see the old way of creating jQuery objects